docs/docs/Installing.md

```diff
@@ -226,7 +226,7 @@ To do so edit `android/build.gradle` and add:
 +            android {
 +                variantFilter { variant ->
 +                    def names = variant.flavors*.name
-+                    if (names.contains("reactNative51") || names.contains("reactNative55")) {
++                    if (!names.contains("reactNative57")) {
```

Comment:
This is the main change where we can simply check if the name does not contain the exact version of you want